scripts/upgrade-v1-test/1_init_ununifid.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# READ ME
+# This script is a test script to check basic operation of upgrade-v1.
+# Note: It is intended to be run in a clean environment.
+# Please be careful not to run it in a production environment or 
+# in an environment where ununifid has already been set up.
+set -e
+cd $HOME
+sudo rm -rf ~/.ununifi
+ununifid init ununifi-upgrade-test-v1 --chain-id ununifi-upgrade-test-v1
+wget https://raw.githubusercontent.com/UnUniFi/network/test/upgrade-v1/test/genesis-mainnet-mock-added-users.json  -O  ~/.ununifi/config/genesis.json;
```

Comment:
We can't use this mock-genesis.json.
If we get error with the previous one, we must modify codes, not genesis.json.

Reply:
You can solve this error by registering an account from bech32 string address before sending tokens or set the account.

Reply:
@taiki1frsy 
> You can solve this error by registering an account from bech32 string address before sending tokens or set the account, I guess.

Do you mean that the account is registered within the upgrade process, rather than manually adding the account to genesis.json?

Reply:
Yes, that's right.
The reason of that is we can't manipulate genesis.json in the stage of upgrade as you understand.
But, this is not heavy problem.
You can do only if you know the bech32 encoded address string! (I believe.)